osu.Game/Input/Bindings/RealmKeyBinding.cs

```diff
@@ -14,7 +16,7 @@ public class RealmKeyBinding : RealmObject, IHasGuidPrimaryKey, IKeyBinding
         [PrimaryKey]
         public Guid ID { get; set; } = Guid.NewGuid();
```

Comment:
#15740 removed the guid primary key from `RealmRulesetSetting` - shouldn't a similar operation happen here too?

Reply:
Ah yeah, will remove it. Don't think it's used (or will ever be used) for anything.

Reply:
Turns out there's one usage of `Find` (which requires a `PrimaryKey`), so I'm going to leave this one as-is in the interest of not making out of scope changes.

https://github.com/ppy/osu/blob/d94b27a8a22bae840d5a1eaac09468946ac8dec7/osu.Game/Overlays/Settings/Sections/Input/KeyBindingRow.cs#L391